src/redux/actions/organisationUnits.js

```diff
@@ -7,23 +7,24 @@ export const ACTION_TYPES = {
     ORGANISATION_UNIT_SELECTED: 'ORGANISATION_UNIT_SELECTED',
 }
 
+export const errorMessage = i18n.t('Could not load organisation units')
+
 export const loadOrganisationUnitsSuccess = periodTypes => ({
     type: ACTION_TYPES.ORGANISATION_UNITS_RECEIVED,
     payload: periodTypes,
 })
 
 export const loadOrganisationUnitsError = () => ({
     type: ACTION_TYPES.ORGANISATION_UNITS_ERRORED,
-    payload: i18n.t('Could not load organisation units'),
+    payload: errorMessage,
 })
 
-export const loadOrganisationUnits = () => dispatch => {
+export const loadOrganisationUnits = () => dispatch =>
     getOrganisationUnits()
         .then(organisationUnits =>
             dispatch(loadOrganisationUnitsSuccess(organisationUnits))
         )
         .catch(() => dispatch(loadOrganisationUnitsError()))
```

Comment:
With the change mentioned above, we could provide the error message to the user:
```.catch(({ message }) => dispatch(loadOrganisationUnitsError(message)))```

Reply:
implemented with a helper function that will render a human readable error message